pdbufr/bufr_read.py

```diff
@@ -21,7 +22,8 @@ def read_bufr(
     columns: T.Iterable[str],
     filters: T.Mapping[str, T.Any] = {},
     required_columns: T.Union[bool, T.Iterable[str]] = True,
-) -> pd.DataFrame:
+    geopandas: bool = False,
```

Comment:
Please add an exception with a clear error message if `geopandas and not HAS_GEOPANDAS`

Reply:
Yes, again - also included